themes/digital.gov/layouts/partials/core/sitewide-alert.html

```diff
@@ -1,31 +1,44 @@
-<!--
-Creates sitewide banner alert at top of page under the banner
--->
+{{ range .Site.Data.sitewide_alert }}
+  {{ $is_emergency := .is_emergency }}
+  {{ $has_icon := .has_icon }}
+  {{ $is_slim := .is_slim }}
+  {{ $header := .header }}
 
-<section class="usa-site-alert usa-site-alert--info" aria-label="Site alert">
-  <div class="usa-alert">
-    <div class="usa-alert__body">
-      <h3 class="usa-alert__heading">
-        Guidance for U.S. government websites and social media for:
-      </h3>
-      <ul class="usa-list">
-        <li>
-          Hawaii wildfires:
-          <a
-            class="usa-link"
-            href="{{- "/resources/hawaii-wildfires-guidance-for-us-government/" | relURL -}}"
-            >digital.gov/hawaii-wildfires</a
-          >
-        </li>
-        <li>
-          Hurricane Idalia:
-          <a
-            class="usa-link"
-            href="{{- "/resources/hurricane-idalia-guidance-for-us-government/" | relURL -}}"
-            >digital.gov/hurricane-idalia</a
-          >
-        </li>
-      </ul>
-    </div>
-  </div>
-</section>
+  {{ if (eq .display_alert true ) }}
+    <section
+      class="usa-site-alert {{ if $is_emergency }}
+        usa-site-alert--emergency
+      {{ else }}
+        usa-site-alert--info
+      {{ end }}{{ if not $has_icon }}
+        usa-site-alert--no-icon
+      {{ end }}{{ if $is_slim }}usa-site-alert--slim{{ end }}"
+      aria-label="Site alert"
+    >
+      <div class="usa-alert">
+        <div class="usa-alert__body">
+          {{ if $header }}
+            <h3 class="usa-alert__heading">{{- $header -}}</h3>
+          {{ end }}
+          {{ if eq (len .alerts) 1 }}
+            <p class="usa-alert__text">
+              {{ index (.alerts) 0 | markdownify }}
+            </p>
+          {{ else }}
+            <ul class="usa-list">
+              {{ range .alerts }}
+                <li>{{ . | markdownify }}</li>
+              {{ end }}
+            </ul>
+          {{ end }}
+        </div>
+      </div>
+    </section>
+  {{ end }}
+{{ end }}
+
+<script type="text/javascript">
+  {{/*  Add the usa-link classes to for consistent link styles  */}}
+  const links = document.querySelectorAll(".usa-alert__body a");
```

Comment:
In the future, it would be better to adjust the data structure so we can build our own links and rely less on JS.

```suggestion
  const links = document.querySelectorAll(".dg-site-alert a");
```

Reply:
Like this idea, implemented!